charts/ingress-nginx/values.yaml

```diff
@@ -583,7 +583,7 @@ controller:
   # -- Modules, which are mounted into the core nginx image. See values.yaml for a sample to add opentelemetry module
   extraModules: []
   # - name: opentelemetry
-  #   image: registry.k8s.io/ingress-nginx/opentelemetry:v20220906-g981ce38a7@sha256:aa079daa7efd93aa830e26483a49a6343354518360929494bad1d0ad3303142e
+  #   image: registry.k8s.io/ingress-nginx/opentelemetry:v20221114-controller-v1.5.1-6-ga66ee73c5@sha256:41076fd9fb4255677c1a3da1ac3fc41477f06eba3c7ebf37ffc8f734dad51d7c
```

Comment:
Should we add this as a boolean as we do with other pieces, we can then have a helm deploy to test it. Right now, it won't be testing in our Helm CI.

Reply:
Thanks, I actually prefer that. Will add it

Reply:
This is ready for review, thanks :)